### PR TITLE
Refactor and cleanup

### PR DESCRIPTION
**Pull request by 50bhan**

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -76,7 +76,7 @@
     | Supported drivers: "cache", "database"
     |
     */
-    'token_storage' => 'database',
+    'token_storage' => 'cache',
```
Comment (mohammad-fouladgar) [inline config/config.php:79]: Why did you change it, what is your reason? And you should update the readme doc, right?

**Review by mohammad-fouladgar — COMMENTED** [review]
Good job! 
Please update the `README.md` file about the `token_storage` config file if we need to change the default driver.

File: config/config.php (hunk 1 above)
Comment (50bhan) [reply config/config.php:79]: I think it's better to set the default driver to `cache` because it's easier and we'd rather use it over `database`. And yes, we need to update README as well. If we're on the same page with this, I'm gonna do the change.

Comment (mohammad-fouladgar) [reply config/config.php:79]: Ok, so after updating readme file please merge this PR. Thanks again 🙏🏻
